src/main/java/mj/chess/Console.java

```diff
@@ -3,37 +3,43 @@
 import java.util.Scanner;
 
 public class Console {
+    private Board board;
+
     public static void main(String[] args) {
         Console console = new Console();
         console.runConsole();
     }
 
     private void runConsole() {
-        Scanner scanner = new Scanner(System.in);
-        String command;
-
-        while (true) {
-            System.out.print("> ");
-            command = scanner.nextLine();
-
-            if (command.equalsIgnoreCase("START")) {
-                startGame();
-            } else if (command.equalsIgnoreCase("END")) {
-                endGame();
-                break;
-            } else System.out.println("Invalid Input");
-        };
 
-        scanner.close();
+        String command;
+        board = new Board();
```

Comment:
여기서 아래와 같이 선언해주는게 나아 보인다는 말씀이었습니다. 

```suggestion
        Board board = new Board();
```

만약 아래 `private` 메소드들때문에 그렇게 설계하지 못하셨다면, `private` 메소드들에서 `Board` 타입의 파라메터를 받아주면 될 것으로 생각합니다.

Reply:
그렇군요! 불필요하게 클래스 멤버변수를 만들어서 자원을 낭비한다는 말씀으로 이해했습니다.

클래스 멤버변수는 스태틱영역(메소드영역)에 계속 남지만, 메소드의 지역변수는 해당 메소드가 스택영역에서 pop되면 함께 제거되기 때문에 자원 관리면에서 더 낫겠네요.

Reply:
@MJbae 자원 관리보다 더 중요한 문제들이 있습니다. 다만 지금 단계에서 다루기는 조금 깊은 내용들이고요, 
단계를 진행해가면서 하나씩 발견해 보시는것도 큰 재미가 될 것 같아서 그 재미를 지금 망치지는 않을게요. 
_왜 그때 그 리뷰어는 이런 말을 했을까?_ 를 항상 유념해 보세요.

Reply:
코드스쿼드의 이런 문화(답을 알려주지 않고 직접 찾게 만드는)는 지적 호기심을 불태우는 것 같습니다ㅎㅎ
본 리뷰는 항상 유념하고 있겠습니다!